CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 - Removed: async mode/option
 ([#107](https://github.com/postcss/postcss-import/pull/107))
+- Removed: bower is not currently supported, use custom resolver
+([#116](https://github.com/postcss/postcss-import/pull/116))
```

Comment:
I know that, but bower is more a downloader, so I guess lots of people just point to the right css file when they import it (eg: `@import "jquery-slider-fx-2016/slider.css"`) so we don't really care about it. The idea is just to inform people that are currently using the feature we are breaking how to upgrade easily.